topic_modeling/scrape_training_data/get_article_links.py

```diff
@@ -17,17 +20,20 @@
 
     for page in pages:
 
-        html_page = urllib.request.urlopen('http://www.thedartmouth.com/search?q=0&page=' + page +
-                                           '&ti=0&tg=0&ty=0&ts_month=0&ts_day=0&ts_year=' + str(year_url) +
-                                           '&te_month=0&te_day=0&te_year=' + str(year_url + 1) + '&s=0&au=0&o=0&a=1')
+        html_page = urllib.request.urlopen(
+            'http://www.thedartmouth.com/search?q=0&page=' +
+            page + '&ti=0&tg=0&ty=0&ts_month=0&ts_day=0&ts_year=' +
+            str(year_url) + '&te_month=0&te_day=0&te_year=' +
+            str(year_url + 1) + '&s=0&au=0&o=0&a=1')
```

Comment:
try to use more of `%` or `.format` way to prepare strings. That would avoid need for manual `str()` calls and make formatting more flexible where needed

Reply:
thanks! fixed here: https://github.com/emtempleton/topic_modeling/pull/6/commits/e2b78762027a209f6fe64adb2854674e91e128bc